component/button/button.css

```diff
@@ -0,0 +1,106 @@
+@import '/index.css';
+
+.btn {
+  border: none;
+  font-weight: 600;
+  padding: 0.5rem 1rem;
+  border-radius: 4px;
+  cursor: pointer;
+  background: transparent;
+}
+
+/** button style **/
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-secondary {
+  background-color: var(--secondary-color);
+  color: var(--primary-text-color);
+}
+
+.btn-accented {
+  background-color: var(--accent-color);
+}
+
+/** outlined button **/
+
+.btn-outline-primary {
+  border: 3px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn-outline-secondary {
+  border: 3px solid var(--secondary-color);
+  color: var(--secondary-color);
+}
+
+.btn-outline-accented {
+  border: 3px solid var(--accent-color);
+  color: var(--secondary-text-color);
+}
+
+/** button with icons **/
+
+.btn-icon {
+  display: inline-block;
+  margin-right: 4px;
+}
+
+/** link button **/
+
+.btn-link-primary {
+  color: var(--primary-color);
+}
+
+.btn-link-secondary {
+  color: var(--secondary-color);
+}
+
+.btn-link-accented {
+  color: var(--accent-color);
+}
+
+/** floating button **/
+
+.btn-float,
+.btn-only-icon {
+  padding: 0;
+  height: 3rem;
+  width: 3rem;
+  border-radius: 50%;
+  font-size: 1.2rem;
+}
+
+/** icon button **/
+
+.btn-only-icon {
```

Comment:
I can remove it and add its width and height to the above class of the same name(btn-only-icon).